en/msg_docs/NeuralControl.md

```diff
@@ -0,0 +1,19 @@
+# NeuralControl (UORB message)
```

Comment:
Please remove this from here and the SUMMARY.md. The docs will be automatically pushed when the source comes in, and having it here now will create a clash later.